src/scene/vcModel.cpp

```diff
@@ -127,6 +131,11 @@ vcModel::vcModel(vcProject *pProject, vdkProjectNode *pNode, vcState *pProgramSt
     pLoadInfo->pModel = this;
     pLoadInfo->pProgramState = pProgramState;
 
+    if (!pNode->hasBoundingBox)
```

Comment:
Should this be `if (pNode->hasBoundingBox)` ?

Reply:
This is correct- if the node _doesn't_ have a bounding box then I've thrown data in the `boundingBox` struct item; if it -does have one (from disk) then I don't want to use those values